pkg/repository/currency_test.go

```diff
@@ -0,0 +1,37 @@
+package repository
+
+import (
+	"luis/goapi/pkg/entity"
+	"luis/goapi/test/testData"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCurrencyRepo_WriteCurrencies(t *testing.T) {
+
+	var testCases = []struct {
+		name            string
+		paramCurrencies []entity.Currency
+		error
+	}{
+		{
+			"Should be succesful",
+			testData.Currencies,
+			nil,
+		}, //Not very sure how to add more test cases
+		/* 	{
+			"Should not be succesful",
```

Comment:
Using TDD implies, that you know which variables need to be parametrized in order to make your testcases, is good to leave comment, but things that may be parametrized this way could be:
- Parameters for your functions on the layer
- If you expect some behavior (should we success on the call or fail?
- If you expect a certain amount of results.